src/jax_state_calc.py

```
@@ -0,0 +1,177 @@
+                state_scores_dict = {}
+                final_states = []
+                state_seq_type = {}
+
+                for start_state_tuple, final_state_tuple in batch_results.items():
+                    start_state = list(start_state_tuple)
+                    final_state = list(final_state_tuple)
+
+                    start_state_str = "".join(map(str, start_state))
+                    final_states.append("".join(map(str, final_state)))
+
+                    state_scores_dict[start_state_str] = self._CellCycleStateCalculation__calculate_state_scores(
```

Pylint (3.3.7)

[convention] Line too long (113/100)

(C0301)